apps/studio.giselles.ai/biome.json

```diff
@@ -1,18 +1,6 @@
 {
-	"$schema": "https://biomejs.dev/schemas/1.8.2/schema.json",
```

Comment:
the current version is 1.9.4 in studio's package.json so it's mismatched. we should refer node_modules, not external URL